### PR TITLE
#296 recover the last uploaded DB Backup gdrive ID and display it for the user

### PR DESCRIPTION
**Pull request by salgadobreno**
#296 
* recover the last uploaded DB Backup gdrive ID and display it for the user
* added puzzle to automate this step

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#349](https://github.com/timelesslounge/timelessis/issues/349) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#349](https://github.com/timelesslounge/timelessis/issues/349) is assigned to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/7fd4d0c4-721c-4e73-b9bf-a3b4b110d937); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -257,9 +257,9 @@ upload, you need to set the `FILE_ID` parameter in `pg_backup.config`, e.g.
 FILE_ID=0B-Iak7O9SfIpYk9zTjZvY2xreVU
 ```
 
-@todo #224:30min We need a more user friendly way to find out the file ID from
- Google Drive. Perhaps do "gdrive list" with the file name as query and set
- file ID from there.
+@todo #296:30min Automate the setting up of FILE_ID into pg_backup.config,
```
Comment (ammaratef45) [inline README.md]: @salgadobreno PDD complain
>puzzle at line #260; Space expected at 261:0; make sure all lines in the puzzle body have a single leading space.

**Comment by ammaratef45** [comment]
@salgadobreno added a comment, check it please.

**Comment by salgadobreno** [comment]
> @salgadobreno added a comment, please check it.

Resolved, thanks!

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 9 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#349](https://github.com/timelesslounge/timelessis/issues/349) is not in the agenda of @ammaratef45[/z](https://www.0crat.com/u/ammaratef45), can't inspect

**Comment by 0crat** [comment]
The job [#349](https://github.com/timelesslounge/timelessis/issues/349) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ammaratef45** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/timelesslounge/timelessis/issues/349#issuecomment-465915185))

@ammaratef45 This is what I know about this job in [CF9LG1YCA](https://www.0crat.com/p/CF9LG1YCA), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#349](https://github.com/timelesslounge/timelessis/issues/349) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka) for 17min
  * @ammaratef45[/z](https://www.0crat.com/u/ammaratef45) will receive 20 minutes after quality review
  * Job [footprint](https://www.0crat.com/footprint/CF9LG1YCA?q=%7Bjob%3A%27gh%3Atimelesslounge%2Ftimelessis%23349%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by ypshenychka** [comment]
@ammaratef45 ccording to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by ammaratef45** [comment]
@ypshenychka the job is deleted from my agenda, zerocrat didn't mention you too :laughing:

**Comment by ammaratef45** [comment]
> @ammaratef45 According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
> 
> > The code reviewer found at least three problems in the code.
> > Comments were mostly about design problems, not cosmetic issues.
> 
> Only one issue was found during code review.
> Please confirm that you'll try to find at least three major problems while future reviews.

Confirm

**Comment by ypshenychka** [comment]
@ammaratef45 thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**